examples/research_projects/dreambooth_inpaint/train_dreambooth_inpaint.py

```diff
@@ -242,6 +242,24 @@ def parse_args():
         ),
     )
     parser.add_argument("--local_rank", type=int, default=-1, help="For distributed training: local_rank")
+    parser.add_argument(
+        "--checkpointing_steps",
+        type=int,
+        default=500,
+        help=(
+            "Save a checkpoint of the training state every X updates. These checkpoints are only suitable for resuming"
```

Comment:
```suggestion
            "Save a checkpoint of the training state every X updates. These checkpoints can be used both as final checkpoints in case they are better than the last checkpoint and are suitable for resuming"
```

The checkpoints should also be useful by themselves no? What couldn't one re-use those for inference?  

We could maybe add to the docs how to use intermediate checkpoints for inference, but it's definitely possible and should also be encouraged (see issue here: https://github.com/huggingface/diffusers/issues/732). People will just have to "build" their pipeline afterwards, but this shouldn't be too hard.

Reply:
Great point, I'll add it to the docs.